### ISSUE TITLE
Dangerous update to AWS Organization Settings

### ISSUE THREAD

**Issue by comjf** [aws-samples/amazon-guardduty-for-aws-organizations-with-terraform#3]
Hello,

This issue is to report dangerous behavior within the `import-org` part of this scripts execution. After following the instructions in this repo, we lost the ability to login to any of our AWS accounts since we use `AWS SSO`.  I traced the problem to an update of the organization settings preformed (via `-auto-approve` no less) which removed SSOs permission to operate in our org.

[This terraform code has a static list of `aws_service_access_principals` defined](https://github.com/aws-samples/amazon-guardduty-for-aws-organizations-with-terraform/blob/main/import-org/org.tf#L22) and overrides the existing organization with these values. This should not be the case. A data-element should be used as a reference to the `aws_organization`, or some kind of querying to ensure this script is only adding to this principals list instead of overriding it.

**Comment by aarthkan** [comment]
Hi,

Thank you for bringing this to attention. The older version of the code had instructions to update this particular file before running the full setup. I see that the instructions were unintentionally removed during a recent update. I will add a quick fix to check this before setting up in the documentation. I will also work on a longer term fix to automate this part, as suggested.

**Comment by aarthkan** [comment]
Hello @comjf,

I have just pushed a new version with changes to automate the update of service access principals. The setup script will now get the list of service access principals and add the one for GuardDuty to the list and then update the Organization settings. Pls let me know if this addresses the issue raised here.
Pls note that I have also made some other updates (not related to this issue) as documented in the Changelog.
Thanks !
